src/main/java/org/opentripplanner/model/plan/ScheduledTransitLeg.java

```diff
@@ -427,6 +433,7 @@ public String toString() {
       .addEnum("alightRule", getAlightRule())
       .addObj("transferFromPrevLeg", transferFromPrevLeg)
       .addObj("transferToNextLeg", transferToNextLeg)
+      .addEnum("realtimeState", getRealTimeState())
```

Comment:
Not sure if we should add this to the toString as this is not Leg's own field.

Reply:
Removed.